commit 70d39a6e62d31a4a7372a712ccc6f8063bbb1550
Author: Vladimir Oltean <vladimir.oltean@nxp.com>
Date:   Fri Jan 15 04:11:16 2021 +0200

    net: mscc: ocelot: export NUM_TC constant from felix to common switch lib
    
    We should be moving anything that isn't DSA-specific or SoC-specific out
    of the felix DSA driver, and into the common mscc_ocelot switch library.
    
    The number of traffic classes is one of the aspects that is common
    between all ocelot switches, so it belongs in the library.
    
    This patch also makes seville use 8 TX queues, and therefore enables
    prioritization via the QOS_CLASS field in the NPI injection header.
    
    Signed-off-by: Vladimir Oltean <vladimir.oltean@nxp.com>
    Reviewed-by: Florian Fainelli <f.fainelli@gmail.com>
    Signed-off-by: Jakub Kicinski <kuba@kernel.org>

diff --git a/include/soc/mscc/ocelot.h b/include/soc/mscc/ocelot.h
index e548b0f51d0c..1dc0c6d0671a 100644
--- a/include/soc/mscc/ocelot.h
+++ b/include/soc/mscc/ocelot.h
@@ -98,6 +98,7 @@
 #define IFH_REW_OP_TWO_STEP_PTP		0x3
 #define IFH_REW_OP_ORIGIN_PTP		0x5
 
+#define OCELOT_NUM_TC			8
 #define OCELOT_TAG_LEN			16
 #define OCELOT_SHORT_PREFIX_LEN		4
 #define OCELOT_LONG_PREFIX_LEN		16